api_tests/base/test_filters.py

```diff
@@ -222,3 +229,51 @@ def test_convert_value_float(self):
         field = FakeSerializer._declared_fields['float_field']
         value = self.view.convert_value(value, field)
         assert_equal(value, 42.0)
+
+
+
+class TestODMOrderingFilter(ApiTestCase):
+    def test_filter_queryset_fails(self):
+        class query:
+            title = ' '
+            def __init__(self, title):
+                self.title = title
+            def __str__(self):
+                return self.title
+        query_s = []
+        q1 = query('NewProj')
+        query_s.append((q1))
+        q2 = query('Zip')
+        query_s.append((q2))
+        q3 = query('Proj')
+        query_s.append((q3))
+        q4 = query('Activity')
+        query_s.append((q4))
+        sort_q = sorted(query_s, cmp=filters.sort_multiple(['-title']))
+        sortt = []
+        for i in sort_q:
+            sortt.append(str(i))
+        assert_equal((sortt) , ['Zip', 'Proj', 'NewProj', 'Activity'])
+    
+    def test_filter_queryset_duplicate_fails(self):
+        class query:
```

Comment:
Since we duplicate this helper class definition, let's move it up into the scope of the containing class, so we can access it as `self.query` in both tests.